### PR TITLE
cmake-nativesdk: install toolchain file for Qt5 bins.

### PR DESCRIPTION
**Pull request by bjorn-oivind**
This installs a cmake file setting the OE_QMAKE_PATH_EXTERNAL_HOST_BINS variable to the location of the native Qt5 binaries in the SDK as given by the OE_QMAKE_PATH_HOST_BINS environment variable. This is necessary to be able to use the SDK to build Qt-applications for the target. See also 05103827d50ec which introduced the error message intended for incorrect usage of the cmake bbclass rather than the cmake_qt5 bbclass.

Please let me know if there is a better way to resolve this, I'm happy to make adjustments if desired.

### CODE REVIEW

**Comment by shr-project** [comment]
Why don't you want to inherit cmake_qt5 in such recipes?

**Comment by bjorn-oivind** [comment]
This is not intended for use with recipes in yocto (for which we already have the cmake_qt5 bbclass), but rather applications developed using the SDK generated by yocto, including the relevant Qt5 libraries and tools. When using CMake to develop applications with the SDK, you cannot currently configure without manually specifying OE_QMAKE_PATH_EXTERNAL_HOST_BINS to the location in the SDK where they are installed, which makes using the SDK a bit harder than necessary in my opinion. :)

**Comment by bjorn-oivind** [comment]
I'll happily adjust the commit message if that was not clear enough, btw.

**Comment by bjorn-oivind** [comment]
Rebased to master.
I cannot change the source branch name without opening a new pull request, so this is still thud confusingly enough.

File: recipes-devtools/cmake/cmake_%.bbappend [hunk 1]
```diff
@@ -0,0 +1,12 @@
+FILESEXTRAPATHS_prepend := "${THISDIR}/${PN}:"
```
Comment (shr-project) [inline recipes-devtools/cmake/cmake_%.bbappend]: Use _class-nativesdk here as well and shouldn't this be BPN?

Comment (kraj) [reply recipes-devtools/cmake/cmake_%.bbappend]: yes this should be BPN

**Comment by mdhgarcia** [comment]
Is this going anywhere? We just ran into this hiccough - would be handy to have it set in the SDK environment setup.

**Comment by shr-project** [comment]
No reply on review comments for many months, closing.

**Pull request closed without merging**